src/parse.coffee

```diff
@@ -33,7 +33,7 @@ parse = (source, callback) ->
       err = new Error(err.message)
 
     if apiElements
-      apiElements = apiElements.toRefract()
+      apiElements = fury.minim.toRefract(apiElements)
```

Comment:
You can get hold of the minim namespace via `fury.minim` and toRefract is namespace not element now. (Serialisation is not coupled in the element models).